gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManager.java

```diff
@@ -125,7 +125,8 @@ void handleTaskEventUpdate(String loggerEventString) {
                 if (optionalTask.isPresent()) {
                     Task task = optionalTask.get();
                     OffsetDateTime offsetDateTime = OffsetDateTime.parse(loggerEvent.getTimestamp());
-                    task.addProcessEvent(offsetDateTime, loggerEvent.getLevel(), loggerEvent.getMessage());
+                    String updatedMessage = loggerEvent.getEventPrefix().isPresent() ? "[" + loggerEvent.getEventPrefix().get() + "] : " + loggerEvent.getMessage() : loggerEvent.getMessage();
```

Comment:
Maybe this method should be in the TaskLogEventUpdate directly ? As it is a prefix, maybe the getMessage() could return directly prefix + message

Reply:
I agree

Reply:
Yes it's better !